drivers/SmartThings/philips-hue/src/lunchbox/rest.lua

```diff
@@ -92,6 +92,8 @@ local function parse_chunked_response(original_response, sock)
         if partial ~= nil and #partial >= 1 then
           full_response:append_body(partial)
```

Comment:
My first thoughts with this were that this was potentially incorrect and that if the socket was closed, then we would always want to return that as an error but that caused some issues with commands while testing on hub.